README.md

```diff
@@ -1,8 +1,8 @@
 # Fivetran Async Provider for Apache Airflow
 
 This package provides an async operator, sensor and hook that integrates [Fivetran](https://fivetran.com) into Apache Airflow.
-`FivetranSensorAsync` allows you to monitor a Fivetran sync job for completion before running downstream processes.
-`FivetranOperatorAsync` submits a Fivetran sync job and polls for its status on the triggerer.
+`FivetranSensor` allows you to monitor a Fivetran sync job for completion before running downstream processes.
+`FivetranOperator` submits a Fivetran sync job and polls for its status on the triggerer.
```

Comment:
Please mention that this point is applicable only when deferrable is set to True

```suggestion
`FivetranOperator` submits a Fivetran sync job and waits for it to complete, on the Airflow worker.When deferrable param is set to True, it would release the Airflow worker slot and poll for the status on the triggerer.
```

Reply:
@phanikumv the FivetranOperator does not wait for the job to complete. It only submits and then we use a sensor to monitor for completion